msteams-platform/samples/app-templates.md

```diff
@@ -210,6 +210,20 @@ E-Prescriptions is a [Power Apps](/powerapps/maker/canvas-apps/embed-teams-app)-
 :::column-end:::
 :::row-end:::
 
+
+## Emergency Button App 
+
+The Emergency Button Power App can be used by organisations who are using Microsoft Teams to enable any set of users to request assistance from supervisors. Features include:
+
+-   Requesting assistance on different categories from a Power App
+-   Notifications sent to requestor informing them of who has been assigned 
+-   Notifications sent to assigned supervisor informing them of who needs assistance 
+-   Audit trail held in SharePoint site
+-   And many more.
```

Comment:
Suggest rephrasing to:
```suggestion
## Emergency Button Power &#9734;

The Emergency Button Power app can be used by organizations that are using Microsoft Teams, to enable any set of users to request assistance from supervisors. This app includes various features, such as:
-   Requesting assistance on different categories from a Power App
-   Notifications sent to requestors informing them of who has been assigned 
-   Notifications sent to assigned supervisors informing them of who needs assistance 
-   Viewing audit trails held in SharePoint
```
Please verify for technical accuracy.


**Questions/verifications**:
1. Please verify the app name
2. Added a star as it is a new app template
3. Modified the overview and the audit trails point 
4. Please re-confirm that no images are required